examples/config.hjson

```diff
@@ -3,8 +3,8 @@
   # https://join-lemmy.org/docs/en/administration/configuration.html
 
   database: {
-    host: postgres
-    password: "{{ postgres_password }}"
+    uri: "postgresql://lemmy:{{ postgres_password }}@postgres/lemmy"
```

Comment:
Maybe add this one as a comment, bc the original `host: postgres` should be good.